File: 2255/main.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println(countPrefixes([]string{"a","a"}, "aa"))
}

func countPrefixes(words []string, s string) int {
	ans := 0
	for _, word := range words {
		f := false
		for i := range word {
			if word > s {
				break
			}
			if word[i] != s[i] {
				f = false
				break
			}
			f = true
		}
		if f {
			ans++
		}
	}
	return ans
}

File: 2255/main.rs
struct Solution{}

impl Solution {
    pub fn count_prefixes(words: Vec<String>, s: String) -> i32 {
        let mut ans = 0;
        for word in words {
            if s.starts_with(&word) {
                ans += 1;
            }
        }
        ans
    }
}

fn main() {
    println!("{}", Solution::count_prefixes(vec!["a".to_string(), "a".to_string()], "aa".to_string()));
}